code/modules/organs/organ_external.dm

```diff
@@ -911,351 +801,205 @@ Note that amputating the affected organ does in fact remove the infection from t
 		H.drop_item()
 	W.loc = owner
 
+/obj/item/organ/external/removed(var/mob/living/user, var/ignore_children)
+
+	var/is_robotic = status & ORGAN_ROBOT
+	..()
+
+	status |= ORGAN_DESTROYED
+	owner.bad_external_organs -= src
+
+	for(var/implant in implants) //todo: check if this can be left alone
+		del(implant)
+
+	// Attached organs also fly off.
+	if(!ignore_children)
+		for(var/obj/item/organ/external/O in children)
+			O.removed()
+			if(O)
+				O.loc = src
+
+	// Grab all the internal giblets too.
+	for(var/obj/item/organ/organ in internal_organs)
+		organ.removed()
+		organ.loc = src
+
+	release_restraints()
+	owner.organs -= src
+	owner.organs_by_name[limb_name] = null // Remove from owner's vars.
+
+	//Robotic limbs explode if sabotaged.
+	if(is_robotic && sabotaged)
+		owner.visible_message(
+			"<span class='danger'>\The [owner]'s [src.name] explodes violently!</span>",\
+			"<span class='danger'>Your [src.name] explodes!</span>",\
+			"<span class='danger'>You hear an explosion!</span>")
+		explosion(get_turf(owner),-1,-1,2,3)
+		var/datum/effect/effect/system/spark_spread/spark_system = new /datum/effect/effect/system/spark_spread()
+		spark_system.set_up(5, 0, owner)
+		spark_system.attach(owner)
+		spark_system.start()
+		spawn(10)
+			del(spark_system)
+		del(src)
+
+/obj/item/organ/external/proc/disfigure(var/type = "brute")
+	if (disfigured)
+		return
+	if(type == "brute")
+		owner.visible_message("\red You hear a sickening cracking sound coming from \the [owner]'s [name].",	\
+		"\red <b>Your [name] becomes a mangled mess!</b>",	\
+		"\red You hear a sickening crack.")
+	else
+		owner.visible_message("\red \The [owner]'s [name] melts away, turning into mangled mess!",	\
+		"\red <b>Your [name] melts away!</b>",	\
+		"\red You hear a sickening sizzle.")
+	disfigured = 1
+
 /****************************************************
 			   ORGAN DEFINES
 ****************************************************/
 
-/datum/organ/external/chest
-	name = "chest"
+/obj/item/organ/external/chest
+	name = "upper body"
+	limb_name = "chest"
 	icon_name = "torso"
-	display_name = "chest"
-	max_damage = 75
-	min_broken_damage = 50
+	health = 100
+	min_broken_damage = 35
 	body_part = UPPER_TORSO
 	vital = 1
-
-/datum/organ/external/groin
-	name = "groin"
+	amputation_point = "spine"
+	gendered_icon = 1
+	cannot_amputate = 1
+	parent_organ = null
+	encased = "ribcage"
+
+/obj/item/organ/external/groin
+	name = "lower body"
+	limb_name = "groin"
 	icon_name = "groin"
-	display_name = "groin"
-	max_damage = 50
-	min_broken_damage = 60
+	health = 100
+	min_broken_damage = 35
 	body_part = LOWER_TORSO
+	vital = 1
```

Comment:
Except for the part where *light* explosions will now have a 5% chance to one-shot you and moderate ones will now have a 22% chance.

Bay's paper spessmen were unpopular here a reason; if the organ is vital *or* explosion code doesn't make an exception to groins, it's going to generate a lot of frustration for our players.

"He threw and IED at me and I died because it dismembered my groin".